pyxrf/model/fit_spectrum.py

```diff
@@ -442,11 +451,14 @@ def update_userpeak(self):
             return
 
         # Ensure, that the values are greater than some small value to ensure that
-        #   there is no computational problems
-        if self.add_userpeak_energy < 0.001:  # 1 eV is very small, the scale is in keV
-            self.add_userpeak_energy = 0.001
-        if self.add_userpeak_fwhm < 0.001:  # 1 eV FWHM is very small, energy axis has 10 eV step
-            self.add_userpeak_fwhm = 0.001
+        #   there is no computational problems.
+        # Energy resolution for the existing beamlines is 0.01 keV, so 0.001 is small
+        #   enough both for center energy and FWHM.
+        energy_small_value = 0.001
+        if self.add_userpeak_energy < energy_small_value:
```

Comment:
just a suggestion:
`self.add_userpeak_energy = max(self.add_userpeak_energy, energy_small_value)`
`self.add_userpeak_fwhm = max(self.add_userpeak_fwhm, energy_small_value)`
might be slightly easier to read

Reply:
Fixed